patches/react-native+0.79.2+011+Add-onPaste-to-TextInput.patch

```diff
@@ -503,12 +503,14 @@ index 42f2383..35bfa4d 100644
 +        if (itemUri != null) {
 +          ContentResolver cr = getReactContext(this).getContentResolver();
 +          type = cr.getType(itemUri);
-+          data = itemUri.toString();
-+          if (mPasteWatcher != null) {
-+            mPasteWatcher.onPaste(type, data);
++          if (type != null && !type.equals(ClipDescription.MIMETYPE_TEXT_PLAIN)) {
```

Comment:
The indentation in this patch seems off, does it matter?

Reply:
Oh yeah the green and red background in the GH diff is confusing on these patches